src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-view-sidebar/external-references/GeneExternalReferences.tsx

```diff
@@ -93,7 +93,17 @@ const buildExternalReferencesGroups = (
     [key: string]: ExternalReferencesGroup;
   } = {};
 
-  externalReferences.forEach((externalReference) => {
+  const sortedExternalReferences = [...externalReferences].sort((a, b) => {
+    if (a.source.name > b.source.name) {
+      return 1;
+    }
+    if (b.source.name > a.source.name) {
+      return -1;
+    }
+    return 0;
+  });
```

Comment:
Isn't it a great case for just using lodash's sortBy? ([docs](https://lodash.com/docs/4.17.15#sortBy))

Reply:
That's done.